AsyncDisplayKit/Details/ASDataController.mm

```diff
@@ -409,9 +451,22 @@ - (void)_reloadDataWithAnimationOptions:(ASDataControllerAnimationOptions)animat
         }
         
         [self _insertSections:sections atIndexSet:[NSIndexSet indexSetWithIndexesInRange:NSMakeRange(0, sectionCount)] withAnimationOptions:animationOptions];
-        
+
         [self _batchLayoutNodes:updatedNodes atIndexPaths:updatedIndexPaths withAnimationOptions:animationOptions];
-        
+
+//        if (_dataSourceImplementsAsyncNodeAtIndexPath) {
```

Comment:
Send me an iMessage when you have another version up for me to review. I would love to help get this in tomorrow, my review is complete and everything looks good to me so far, but I am going to hold off until these comments are corrected and it is more clear what the intended code path is in this critical area. @rahul-malik